en/docs/setup/environment-compatibility.md

```diff
@@ -112,15 +111,18 @@ WSO2 Identity Server supports the following DBMSs.
 ### Tested Web Browsers
 
 <ul>
-<li>Chrome Version 78.0.3904.87 (Official Build) (64-bit)</li>
-<li>Firefox 70.0.1 (64-bit)</li>
-<li>Internet Explorer 11.576.14393</li>
-<li>Safari 13.0.3</li>
+	<li>Mozilla Firefox 75.0 (64-bit)</li>
+	<li>Mozilla Firefox 81.0.2 (64-bit)</li>
+	<li>Google Chrome Version 84.0.4147.125 (Official Build) (64-bit)</li>
+	<li>Google Chrome Version 85.0.4183.102 (Official Build) (64-bit)</li>
+	<li>Google Chrome Version 86.0.4240.75 (Official Build) (x86_64)</li>
+	<li>Chromium version 86.0.4240.111 (Official Build)</li>
+	</li>MS Edge 44.18362</li>
 </ul>
 
 ### Tested LDAPs
 
-Following is a list of LDAPs that WSO2 Identity Server 5.9.0 is tested with.
+Following is a list of LDAPs that WSO2 Identity Server 5.11.0 is tested with.
 
 <ul>
 	<li>Open LDAP 2.4.28</li>
```

Comment:
Please Add Microsoft Active Directory Windows 2012

Reply:
This isn't removed.